build.js

```diff
@@ -52,51 +43,45 @@ StyleDictionary.registerTransform({
   },
 });
 
+const customColorObjectFormatter = (dictionary, isJS) => {
+  const valueOrType = (token) =>
+    isJS ? `"${token.value}"` : `${getTypeScriptType(token.value)}`;
+  const declaration = isJS ? "" : `export const `;
```

Comment:
This could just be `isJS && "export const"`, no?

Reply:
i wanted to but unfortunately that returns this: 🥲

```
module.exports = {
  falsegreen: {
    100: "#FAFFFC",
    200: "#EBF2EF",
    300: "#D1E0D9",
```

Reply:
Awww, I guess that makes sense! Damn javascript string interpolation